app/frontend/src/App.vue

```diff
@@ -8,6 +8,7 @@
       <transition name="component-fade" mode="out-in">
         <router-view />
       </transition>
+      <FirstTimeUserInfoBanner />
```

Comment:
If the form is public, it won't because a public form does not go through login. If they have to log in, it will but they can close the dialog. I can set up a meeting to discuss it.